Install/README.md

```diff
@@ -31,23 +31,42 @@ An example using options appended to the command can be:
 curl -kL dexterindustries.com/update_gopigo3 | bash -s --user-local --no-update-aptget --no-dependencies
 ```
 
-#### Command Options
+## Command Options
 
 The options that can be appended to this command are:
 
+* `--no-update-aptget` - to skip using `sudo apt-get update` before installing dependencies. For this to be useful, `--no-dependencies` has to be not used. Applies to RFR_Tools and the GoPiGo3.
+* `--bypass-rfrtools` - skips installing RFR_Tools completely.
+    * `--bypass-python-rfrtools` - skips installing/updating the python package for  [RFR_Tools](https://github.com/DexterInd/RFR_Tools).
+    * `--bypass-gui-installation` - skips installing the GUI packages/dependencies from [RFR_Tools](https://github.com/DexterInd/RFR_Tools).
 * `--no-dependencies` - skip installing any dependencies for the GoPiGo3. It's supposed to be used on each consecutive update after the initial install has gone through.
-* `--no-update-aptget` - to skip using `sudo apt-get update` before installing dependencies. For this to be useful, `--no-dependencies` has to be not used.
-* `--bypass-rfrtools` - skip installing RFR_Tools altogether.
-* `--bypass-python-rfrtools` - skips installing/updating the python package for  [RFR_Tools](https://github.com/DexterInd/RFR_Tools).
 * `--user-local` - install the python package for the GoPiGo3 in the home directory of the user. This doesn't require any special read/write permissions: the actual command used is (`python setup.py install --force --user`).
 * `--env-local` - install the python package for the GoPiGo3 within the given environment without elevated privileges: the actual command used is (`python setup.py install --force`).
 * `--system-wide` - install the python package for the GoPiGo3 within the sytem-wide environment with `sudo`: the actual command used is (`sudo python setup.py install --force`).
 
 Important to remember is that `--user-local`, `--env-local` and `--system-wide` options are all mutually-exclusive - they cannot be used together.
 As a last thing, different versions of it can be pulled by appending a corresponding branch name or tag.
 
+## Minimal Installation
+
+Now, if you only want the absolute minimum in order to get going with the GoPiGo3, you can run this command:
+```bash
```

Comment:
That's the syntax highlighting feature. You can select any language you want and you'll get the appropriate highlighting. In this case, it's bash, because we're doing it in the shell.